packages/governance/src/contractGovernor.js

```diff
@@ -90,10 +91,15 @@ const validateQuestionFromCounter = async (zoe, electorate, voteCounter) => {
  * up to the contract.
  *
  * The contractGovernor creatorFacet includes voteOnParamChanges(),
- * which is used to create questions that will automatically update
- * contract parameters if passed. This facet will usually be closely held. The
- * creatorFacet can also be used to retrieve the governed instance, publicFacet,
- * and its creatorFacet with voteOnParamChanges() omitted.
+ * voteOnFilter, and voteOnApiInvocation. voteOnParamChanges() is used to create
+ * questions that will automatically update contract parameters if passed.
+ * voteOnFilter can be used to crate questions that will prevent the exercise of
```

Comment:
s/crate/create

Reply:
the questions don't prevent the exercise, right? consider something like "create questions that when affirmed will prevent…"